src/elements/app-bar/styles.css

```diff
@@ -0,0 +1,37 @@
+:root {
+  --app-bar-padding-x: var(--component-padding-x, 1rem);
+  --app-bar-padding-y: var(--component-padding-y, 1rem);
+  --app-bar-padding: var(--app-bar-padding-y) var(--app-bar-padding-x);
+  --app-bar-background: var(--white, #fff);
+  --app-bar-color: var(--gray-800, inherit);
+  --app-bar-hover-background: #0000000a;
+}
```

Comment:
Isn't it possible to define these variables only on module scope level?

Reply:
It's possible but this would be too much. I do not want to do it now. Should we do so? If yes, why?

Reply:
I was just curious. It is not necessary. I just think we have a lot of global variables and it can cause troubles in the future.

Reply:
Not really. It allows you config the framework even if you include configuration after source files. I like it more.

In other words, if you do the following:

```css
@import url('~@oacore/design/lib/index.css');

:root {
  --app-bar-background: var(--gray-100);
}
```

the AppBar will become light grey.

Reply:
Some of the variables are useful some not. I can imagine changing background color but I do not really care about padding you know.